src/reducers/form.js

```diff
@@ -34,16 +34,9 @@ const field = (state = initialFieldState, action) => {
   }
 };
 
-const initialPageState = {
-  show: false,
-};
-
-const page = (state = initialPageState) => ({
-  show: !state.show,
-});
-
+// TODO Hydrate initial form state from query string
 const initialFormState = {
-  pages: {},
```

Comment:
You may want to consider keeping this reducer structure (since this form could be used by other kinds of forms). Then you could say that the 'page' is just hardcoded to 'global' for this form (since it shares global state). If we have forms in the future, we might not want the same form to share global state.